public/src/app/applications/details-panel/details-panel.component.ts

```diff
@@ -56,28 +56,27 @@ export class DetailsPanelComponent implements OnDestroy, OnInit {
     private projectService: ProjectService,
     private spatialFeatureService: SpatialFeatureService,
     private attachmentService: AttachmentService,
-    private fss: FeatureSelectService
+    private fss: FeatureSelectService,
   ) {}
 
   ngOnInit(): void {
     // Note, can't seem to get stateService.ts to get codeTable working here. Instead, subscribe to it.
+    // Subscribe to this first, seems to be slower and can cause minor page render issue due to no code.
     this.projectService.workflowStateCodeControllerFindAll()
     .pipe(take(1)).subscribe((data) => {
       this.workflowStatus = _.keyBy(data, 'code');
     });
-    
-    // subscribe and watch for URL param changes
+
+    // First time component init. The `urlService.onNavEnd$` already ends, so 
+    // do this initially first since queryParam is ready from route. 
+    // Works if user has bookmarks the detail link.
+    this.getProjectDetails();
+
+    // Subscribe to onNavEnd so the component knoews subsequent clicks on other details.
```

Comment:
I wonder if "knoews" is a small typo? thanks!!

Reply:
Thanks for catching that, fixed :)